src/makeRobot.js

```diff
@@ -38,7 +38,64 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
If `keyName === variableName` you can use shortened syntax
```suggestion
    name,
    wheels,
    version,
```